src/applications/claims-status/tests/e2e/02.claim-files.v2.cypress.spec.js

```diff
@@ -520,52 +526,6 @@ describe('Type 1 Unknown Upload Errors', () => {
       .click();
   };
 
-  const getFileInputElement = (fileIndex = 0) =>
```

Comment:
I moved these to a shared file called `src/applications/claims-status/tests/e2e/file-upload-helpers.js`